meshtal_analysis.py

```diff
@@ -209,11 +199,27 @@ def pick_point(x, y, z, mesh, erg):
     return v
      
 
-
-# TODO:
-def add_mesh(mesh1, mesh2):
-    """ """
-    print("not ready yet")
+def add_mesh(mesh1, mesh2): 
+    """ checks if boundaries of two meshes are equal and adds their values and errors  """
+    if mesh1.x_bounds != mesh2.x_bounds and mesh1.y_bounds != mesh2.y_bounds and mesh1.z_bounds != mesh2.z_bounds:
+        raise ValueError('bounds not equal')
+    
+    else:
+        new_val = mesh1.data['value'] + mesh2.data['value']
+        new_err = np.sqrt((mesh1.data['rel_err'])**2+(mesh2.data['rel_err'])**2)
+        
+        new_mesh = meshtally()
+        cols = ("Energy","x", "y", "z", "value", "rel_err")
+        new_mesh.data = pd.DataFrame(columns = cols)
+        new_mesh.ctype = "6col"
+        new_mesh.x_bounds = mesh1.x_bounds
+        new_mesh.y_bounds = mesh1.y_bounds
+        new_mesh.z_bounds = mesh1.z_bounds

```

Comment:
might be good to also generate the xmids, ymids etc for the mesh object so any subsequent operation on the returned mesh doesn't give an error